lib/cylc/review.py

```diff
@@ -661,6 +661,8 @@ def viewsearch(self, user, suite, path=None, path_in_tar=None, mode=None,
     def view(self, user, suite, path, path_in_tar=None, mode=None,
              no_fuzzy_time="0"):
         """View a text log file."""
+        if re.search(r"log\..*\d\d\s\d\d$", path):
+            path = path.replace(' ', '+')
         # get file or serve raw data
```

Comment:
This doesn't work for timezones with minutes (e.g. `+0430`)

Also I think `re.sub` is better suited to this
```suggestion
        # Log files with +TZ in name end up with space instead of plus sign, so
        # put plus sign back in (https://github.com/cylc/cylc-flow/issues/4260)
        path = re.sub(r"(log\.\S+\d{2})\s(\d{2,4})$", r"\1+\2", path)
        
        # get file or serve raw data
```

Reply:
Sound.